.github/actions/build/action.yaml

```diff
@@ -60,3 +60,17 @@ runs:
           path: |
             src/out/${{ matrix.platform }}_qa/apks/*.apk
             src/out/${{ matrix.platform }}_qa/*_apk/*.apk
+      - name: Create Test Files Archive
+        if: startsWith(matrix.platform, 'linux') && matrix.config == 'devel'
+        run: |
+          set -x
+          cd src
+          tar cvf - out/${{ matrix.platform }}_${{ matrix.config }}/*tests out/${{ matrix.platform }}_${{ matrix.config }}/*.so out/${{ matrix.platform }}_${{ matrix.config }}/starboard/*.so | xz -T0 -1 -z - > test_artifacts.tar.xz
```

Comment:
>out/${{ matrix.platform }}_${{ matrix.config }}/starboard/*.so

These are not the file you are looking for. The filename is `libstarboard.so.17` so the glob fails to match.

Change the starboard glob to `*.so.*`.

Reply:
Done.